invenio/modules/authors/recordext/fields/author_base.cfg

```diff
@@ -65,7 +65,7 @@ name:
          'schema': {'last': {'type': 'string', 'empty': False},
                     'first': {'type': 'string', 'empty': False}}}}
     creator:
-        marc, "100__", {'last': util_split(value['a'], ', ', 0),
+        marc, "100._", {'last': util_split(value['a'], ', ', 0),
```

Comment:
> Whats the difference between `._`, `_.`, `..` and `__`?

:black_joker: @jalavik  are we talking morse code - `A` `N` `I` and `M` :wink: ?